lib/gocd/pipeline_status/pipeline.rb

```diff
@@ -13,7 +13,7 @@ def last_build_status
     end
 
     def status
-      {pipeline: name, status: last_build_status}
+      {pipeline: name, status: current_status}
```

Comment:
It was last_build_status because there was a requirement where I wanted to check what is the status of the build in terms of failure or success. In case the build is running, it will return status of the last build.
However I agree that we should have current build status as well. In this case can we return both the values, the status of the last build and current build?

Another thought, maybe we can have 2 separate methods, one will return the current status of the pipeline and another can return the last status irrespective of build is currently running or sleeping.

What do you think?

Reply:
Just noticed that `red?` and `green?` are still using last build status, in that case I think I am ok with  this implementation.